crates/ubrn_cli/src/workspace.rs

```diff
@@ -6,15 +6,22 @@
 use anyhow::Result;
 use camino::Utf8PathBuf;
 
-pub(crate) fn package_json() -> Result<Utf8PathBuf> {
+use crate::config::PackageJson;
+
+fn package_json_file() -> Result<Utf8PathBuf> {
     let pwd = ubrn_common::pwd()?;
 
     let package_json = ubrn_common::resolve(pwd, "package.json")?;
     Ok(package_json.expect("Must be run under a directory containing a package.json file"))
 }
 
+pub(crate) fn package_json() -> PackageJson {
+    let file = package_json_file().expect("Cannot file package.json");
```

Comment:
should this be "Cannot find package.json" instead of *file?

Reply:
The `expect` causes a `panic` if `package_json_file()` returns with an error.